fdbsyncd/fdbsync.cpp

```diff
@@ -0,0 +1,603 @@
+#include <string>
+#include <netinet/in.h>
+#include <netlink/route/link.h>
+#include <netlink/route/neighbour.h>
+#include <netlink/route/link/vxlan.h>
+#include <arpa/inet.h>
+
+#include "logger.h"
+#include "dbconnector.h"
+#include "producerstatetable.h"
+#include "ipaddress.h"
+#include "netmsg.h"
+#include "macaddress.h"
+#include "exec.h"
+#include "shellcmd.h"
+#include "fdbsync.h"
+#include "warm_restart.h"
+#include "errno.h"
+
+using namespace std;
+using namespace swss;
+
+#define VXLAN_BR_IF_NAME_PREFIX    "Brvxlan"
+
+FdbSync::FdbSync(RedisPipeline *pipelineAppDB, DBConnector *stateDb) :
+    m_fdbTable(pipelineAppDB, APP_VXLAN_FDB_TABLE_NAME),
+    m_imetTable(pipelineAppDB, APP_EVPN_REMOTE_VNI_TABLE_NAME),

```

Comment:
According to sonic-swss-common [PR339](https://github.com/Azure/sonic-swss-common/pull/339), should this constant be renamed to `APP_VXLAN_REMOTE_VNI_TABLE_NAME`

Reply:
Thanks for review. Updated the same